src/app/dataset-view/additional-components/metadata-component/components/add-polling-source/add-polling-source.component.html

```diff
@@ -38,11 +38,14 @@
             <form [formGroup]="pollingSourceForm">
                 <ng-container *ngIf="currentStep === steps.FETCH">
                     <app-base-step
+                        *ngIf="history"
```

Comment:
Question why only this step has `history` condition, and not the others.
If we need this, it probably should be set for entire div with forms.